src/mlio/record_readers/csv_record_reader.h

```diff
@@ -43,6 +43,8 @@ class csv_record_reader final : public text_record_reader {
         : text_record_reader{std::move(strm)}, params_{&prm}
     {}
 
+    static const std::size_t MAX_DATA_IN_EXCEPTION = 1024;
```

Comment:
The "modern" way of defining constants is using `static constexpr` instead of `static const`. Also it is discouraged to use all uppercase variable names as they resemble macros. In short this can be defined like:
```
static constexpr std::size_t max_data_in_exception = 1024;
```